inst/sensorreading/sensorreading.Rmd

```diff
@@ -62,8 +62,11 @@ if(!identical(tbl, "The dataset requested is empty.")){
 ***
 
 ```{r,echo=FALSE,result='asis',comment=NA,warning=FALSE}
+library(dplyr)
```

Comment:
it works for functions specifically, so that function can use it, but since the function calls an .Rmd, it starts in a fresh environment. So keep `library(dplyr)` and remove `importFrom`